minilog.cpp

```diff
@@ -96,6 +100,21 @@ struct ThreadLogContext
 	bool hasLogsOnThisLevel[kMaxProcsNesting] = { false };
 };
 
+#if OS_MACOS
+static os_log_type_t logLevelToOsLogType(minilog::eLogLevel level) 
+{
+	switch (level) 
+	{
+		case minilog::eLogLevel::Paranoid: return OS_LOG_TYPE_INFO;
```

Comment:
`Paranoid` is more verbose than `Debug`. Is it the same semantics with `OG_LOG_TYPE_INFO` and `OG_LOG_TYPE_DEBUG`? Which one is more verbose?